.ci/matrix-unix-linux-distros.yml

```diff
@@ -0,0 +1,19 @@
+# This file is used as part of a matrix build in Jenkins where the
+# values below are included as an axis of the matrix.
+
+# This axis of the build matrix represents the Linux distributions on
+# which Logstash will be tested.
+
+os:
+  - amazon
```

Comment:
Can we reduce the scope of this to a debian flavor + redhat flavor, eg centos8+ubuntu-18.04

Reply:
I've separated the compatibility axis (with the full list) and normal test axis with centos 7 and ubuntu 18.04

Reply:
I'm wondering if we should also have a reduced set of OS's in the compatibility axis until we have confidence that the tests pass consistently, and/or we can set these tests to not spam slack and email until we do. cc @roaksoax

Reply:
at the moment we could keep the older version for each distribution (to lower the number of runs) and probably exclude the oraclelinux since it hasn't docker inside

Reply:
I think we should reduce this for the time being to:

   - amazon
   - centos&&immutable
   - debian&&immutable
   - fedora-29&&immutable
   - opensuse-15-1&&immutable
   - oraclelinux&&immutable
   - ubuntu&&immutable